packages/performance/src/definitions.ts

```diff
@@ -105,3 +148,144 @@ export interface IsEnabledResult {
    */
   enabled: boolean;
 }
+
+export interface PutAttributeOptions {
+  /**
+   * Name of the trace to set its attribute.
+   *
+   * @since 6.3.0
+   */
+  traceName: string;
+  /**
+   * Name of the attribute to set its value.
+   *
+   * @since 6.3.0
```

Comment:
```suggestion
   * @since 6.3.0
   * @example "experiment"
```

Feel free to add some example values from time to time if this makes sense.